controlplane/kubeadm/controllers/kubeadm_control_plane_controller_test.go

```diff
@@ -1678,494 +1698,6 @@ func machine(name string) *clusterv1.Machine {
 	}
 }
 
-func TestKubeadmControlPlaneReconciler_failureDomainForScaleDown(t *testing.T) {
-	tests := []struct {
```

Comment:
@chuckha These tests look useful, are we planning to add them later?

Reply:
The code under test was removed. Which part do you think we should consider adding later?

Reply:
failureDomainForScaleDown and failureDomainForScaleUp functions do not have tests anymore, are they being tested somewhere else?

Reply:
Both of those functions were removed. But I see what you mean now, yes, I plan to add tests for their replacements, it will be on the ControlPlane object

Reply:
Ah I didn't see that the real functions are gone. Please ignore.